petpals/templates/_layout.html

```diff
@@ -88,6 +88,12 @@
           <!-- Right elements -->
           <div class="d-flex align-items-center">
             {% if current_user.is_authenticated %}
+            <a
+              class="nav-link{{' active' if active_page == 'messages'}} text-white mt-2"
+              href="{{ url_for('message_router.render_message_page') }}"
+            >
+              Messages
```

Comment:
```html
<i class="fa-solid fa-comment-dots fa-2xl"></i>
```
https://fontawesome.com/search?q=chat&s=solid%2Cbrands
https://fontawesome.com/docs/web/style/size#relative-sizing

You can choose another icon if you want.